Zend/zend_compile.h

```diff
@@ -432,6 +443,10 @@ union _zend_function {
 		uint32_t num_args;
 		uint32_t required_num_args;
 		zend_arg_info *arg_info;
+        struct {
+            zend_uchar type; // the accessor type (getter or setter)
+            uint32_t property_offset;
+        } accessor;
```

Comment:
I suppose, we don't need this in "common". These fields are specific for op_array(s).